react-stopwatch/my-app/src/StopWatch.tsx

```diff
@@ -0,0 +1,43 @@
+import { useState } from 'react';
+import './App';
+import './StopWatch.css';
+import { FaPlay } from 'react-icons/fa';
+import { FaPause } from 'react-icons/fa6';
+
+export function StopWatch() {
+  const [elapsedTime, setElapsedTime] = useState(0);
+  const [intervalID, setIntervalID] = useState<NodeJS.Timeout>();
+
+  function handleClickIcon() {
+    if (intervalID === undefined) {
+      const i = setInterval(() => {
+        setElapsedTime((prev) => prev + 1);
+      }, 1000);
+      setIntervalID(i);
+    } else {
+      clearInterval(intervalID);
+      setIntervalID(undefined);
+    }
+  }
+
+  function handleClickFace() {
+    setElapsedTime(0);
```

Comment:
You should only do this if the stopwatch isn't running (that is, intervalId is undefined).

Reply:
okay! I think I fixed the issue